src/lib/ai/bias-detection/python/bias_detection_service.py

```
@@ -551,19 +531,15 @@ async def _run_model_level_analysis(
             ),
         }
 
-    async def _run_interactive_analysis(
-        self, session_data: SessionData
-    ) -> Dict[str, Any]:
+    async def _run_interactive_analysis(self, session_data: SessionData) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)